sensors/simsensor_common/event_queue.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Queue for buffering sensor event data from reader to simsensors
+ *
+ * Copyright 2023 Phoenix Systems
+ * Author: Piotr Nieciecki
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#ifndef EVENT_QUEUE_H
+#define EVENT_QUEUE_H
+
+#include <stdbool.h>
+#include <time.h>
+
+#include <libsensors.h>
+
+
+typedef struct {
+	sensor_event_t *queue;
+	unsigned int capacity; /* queue max length */
+	unsigned int len;      /* current length of queue */
+
+	sensor_event_t *head;
+	sensor_event_t *tail;
+} event_queue_t;
+
+
+/* Initiates `q` as an event queue. If succeeded returns 0. In other case -1. */
+extern int eventQueue_init(event_queue_t *q, unsigned int cnt);
```

Comment:
This function is never used, seem like an oops

Reply:
Function will be used in modules called `simsensors`, which will be added in the future. Without this function you cannot use whole queue

Reply:
Ack